getting-started/jdbc/README.md

```diff
@@ -40,6 +40,8 @@ This example requires `jq` to be installed on your machine.
     export QUARKUS_DATASOURCE_JDBC_URL=jdbc:postgresql://postgres:5432/POLARIS
     export QUARKUS_DATASOURCE_USERNAME=postgres
     export QUARKUS_DATASOURCE_PASSWORD=postgres
+    export CLIENT_ID=root
+    export CLIENT_SECRET=s3cr3t
```

Comment:
Could you give some more details about why this is necessary?

Reply:
+1, the spark client [here](https://github.com/apache/polaris/blob/main/getting-started/jdbc/docker-compose.yml#L85) passed creds here, can you please share what your command is to start docker